test/simpleweightedgraph.jl

```diff
@@ -329,4 +330,18 @@ using SimpleWeightedGraphs
         @test g[1, 3, Val{:weight}()] ≈ 0
         @test g[2, 3, Val{:weight}()] ≈ 0.5
     end
+
+    # this testset was implemented for https://github.com/JuliaGraphs/SimpleWeightedGraphs.jl/issues/32
+    @testset "induced_subgraph should preserve weights for edge lists" begin
+        g = SimpleWeightedGraph([0 2; 2 0])
+        graphWeights = weights(g)
+        expectedGraphWeights = sparse([0 2; 2 0])
+        @test graphWeights == expectedGraphWeights
+        # vertex induced subgraph
+        vertexInducedSubgraphWeights = weights(first(induced_subgraph(g, [1, 2])))
```

Comment:
Is this already tested elsewhere?

Reply:
I don't think so. I did a search for `induced_subgraph` and this is the only test in this package that accesses this function.